gatsby-browser.js

```diff
@@ -6,4 +6,24 @@
 
 import wrapPageElement from './gatsby/wrap-page-element';
 
-export { wrapPageElement };
+const onRouteUpdate = ({ location }) => {
+  if (process.env.NODE_ENV !== `production` || typeof ga !== `function`) {
+    return null;
+  }
+
+  // wrap inside a timeout to make sure react-helmet is done with it's changes (https://github.com/gatsbyjs/gatsby/issues/9139)
+  // reactHelmet is using requestAnimationFrame: https://github.com/nfl/react-helmet/blob/5.2.0/src/HelmetUtils.js#L296-L299
+  const sendPageView = () => {
+    const pagePath = location
+      ? location.pathname + location.search + location.hash
```

Comment:
Will we always have these properties on the `location` object? Just wanna make sure we're not adding `undefined` to the path.

Reply:
I believe so yes, this is used in gatsby's google analytics plugin